app/views/queue_items/index.html.haml

```diff
@@ -4,27 +4,31 @@
       %article
         %header
           %h2 My Queue
-        %table.table
-          %thead
-            %tr
-              %th(width="10%") List Order
-              %th(width="30%") Video Title
-              %th(width="10%") Play
-              %th(width="20%") Rating
-              %th(width="15%") Genre
-              %th(width="15%") Remove
-          %tbody
-            - @queue_items.each do |item|
+        = bootstrap_form_tag url: '/update_queue' do |f|
```

Comment:
You should use named routes. That way if you decide to change the url for your route, you don't have to update any view templates.